modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -20,7 +20,8 @@ const defaultProps = {
   maxCacheSize: null,
   maxCacheByteSize: null,
   refinementStrategy: STRATEGY_DEFAULT,
-  zRange: null
+  zRange: null,
+  getHighlightedObjectIndex: {type: 'function', value: () => -1, compare: false}
```

Comment:
This is not documented.

If this is going to be a public API, we need to look into whether it's sufficient to just pass it the `tile` argument.

I think it's ok to not expose this until we find other use cases.

Reply:
Yeah, makes sense. Let's remove it and wait until we find some other use cases.